src/utils.js

```diff
@@ -113,14 +120,34 @@ const setupDVC = async opts => {
   }
 
   if (platform === 'win32') {
-    console.log('Installing DVC with pip');
-    console.log(
-      await exec(
-        `pip install --upgrade dvc[all]${
-          version !== 'latest' ? `==${version}` : ''
-        }`
-      )
-    );
+    if (arch === 'x64') {
+      try {
+        const dvcURL = `https://dvc.org/download/win/dvc-${version}`;
+        console.log(`Installing DVC from: ${dvcURL}`);
+        await download(dvcURL, 'dvc.exe');
+      } catch (err) {
+        console.log('DVC Download Failed, trying from GitHub Releases');
+        const dvcURL = `https://github.com/iterative/dvc/releases/download/${version}/dvc-${version}.exe`;
+        console.log(`Installing DVC from: ${dvcURL}`);
+        await download(dvcURL, 'dvc.exe');
+      }
+      console.log(
+        await exec(
+          `powershell -c "Start-Process -FilePath .\\dvc.exe -ArgumentList '/SP- /NORESTART /SUPPRESSMSGBOXES /VERYSILENT' -NoNewWindow -Wait"`
+        )
+      );
+      await fsPromises.unlink('dvc.exe');
+      core.addPath('C:\\Program Files (x86)\\DVC (Data Version Control)');
```

Comment:
I am not sure why this is necessary. Locally, installer adds to the path automatically.